components/identities/availableIdentities.tsx

```
@@ -174,7 +174,7 @@ const AvailableIdentities = ({ tokenId }: { tokenId: string }) => {
           <div>
             <IdentitiesSkeleton />
           </div>
-        ) : !isUpdatingPp ? (
+        ) : false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)